### PR TITLE
[EdgeTPU] Add files for .edgetpucfg

### PR DESCRIPTION
**Pull request by hohee-hee**
Related to #1621,

for modularization of EdgeTPU, we decided to add a new file name extension for config file for edgetpu compiler.
Its name is 'edgetpucfg'.

'edgetpucfg' has different steps and options than the original cfg file.
Therefore, this commit adds a new directory and files for `.edgetpucfg` in /media/EdgetpuCfgEditor.

ONE-vscode-DCO-1.0-Signed-off-by: hohee-hee [khappy517@gmail.com](mailto:khappy517@gmail.com)

### CODE REVIEW

**Comment by jyoungyun** [comment]
Please clean up your commits. :)

**Comment by dayo09** [comment]
I will review this PR when the rest of parts are merged so that the panel can be opened in UI.

File: media/EdgeTPUCfgEditor/vscodeapi.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (Hyeon-Uk) [inline media/EdgeTPUCfgEditor/vscodeapi.js:2]:
```suggestion
 * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
```

i think this line should be changed

**Review by Hyeon-Uk — CHANGES_REQUESTED** [review]

**Comment by dayo09** [comment]
image

👍 

I checked these working well:
- Changing 'Input Path'
  - makes the same change on text viewer.
  - fills 'Output Path' accordingly
- Filling in Intermediate_tensors / Checking 'Show Operations'
  - makes the same change on text viewer
- Checking 'Search Delegate'
  - makes the same change on text viewer
  - shows 'Delegate Search Step' well

**Review by dayo09 — APPROVED** [review]
LGTM, good works :-D Thanks!

**Review by Hyeon-Uk — APPROVED** [review]
Good :-)

**Review by rhqjatn2398 — APPROVED** [review]
LGTM 👍

**Pull request merged**